module/VuFind/tests/integration-tests/src/VuFindTest/Mink/RecordCoverImageTest.php

```diff
@@ -186,6 +186,10 @@ public function testCoverLoading(
             . ($ajaxcovers ? ' ajax' : '')
             . (empty($noCoverAvailableImage) ? ' hidden' : '');
         $coverImage = $this->findCss($page, 'img.recordcover');
+        $session->wait(
+            $this->getDefaultTimeout(),
+            'document.querySelector("img.recordcover").classList = "' . $expectedClasses . '"'
```

Comment:
Here, the `classList` gets assigned and not checked for equality. So that statement is immediately true and the element gets the expected class.